src/components.scss

```diff
@@ -1,5 +1,5 @@
 :host {
-  display: block;
+  display: var(--display);
```

Comment:
It seems like the problem with IE caused by this line. Somehow when we have css vars in global component scss, it will render unique class name for IE styling (such as sc-c-header-0).